### PR TITLE
Replace the deprecated `max_named` and `min_named` methods with `named_args`

### PR DESCRIPTION
**Pull request by WavinFlag**
The methods `max_named` and `min_named` had been deprecated in [brew v3](https://github.com/Homebrew/brew/blob/3.0.0/Library/Homebrew/cli/parser.rb#L396-L405).

When I run `brew alias` it warns me:
> Warning: Calling `max_named` is deprecated! Use `named_args max:` instead.
> Please report this issue to the homebrew/aliases tap (not Homebrew/brew or Homebrew/core), or even better, submit a PR to fix it:
>   /usr/local/Homebrew/Library/Taps/homebrew/homebrew-aliases/cmd/alias.rb:19

### CODE REVIEW

File: cmd/alias.rb
Comment (Rylan12) [inline cmd/alias.rb:19]:
```suggestion
      named_args max: 1
```

Not technically needed but it's more consistent with the rest of the `brew` commands.

Also, can you update lines 11-15 to be:

```ruby
      usage_banner "`alias` [<alias> ... | <alias>=<command>]"
      description <<~EOS
        Show existing aliases. If no aliases are given, print the whole list.
      EOS
```

File: cmd/unalias.rb
Comment (Rylan12) [inline cmd/unalias.rb:16]:
```suggestion
      named_args :alias, min: 1
```

And change the `usage_banner` method to be :

```ruby
       description <<~EOS
         Remove aliases.
       EOS
```

This allows the usage string to be automatically generated

**Review by Rylan12 — COMMENTED** [review]
Thanks, @WavinFlag!

A few suggestions

**Comment by WavinFlag** [comment]
Sorry for the delay.
Now it should be all ok.

File: cmd/unalias.rb
Comment (WavinFlag) [reply cmd/unalias.rb]: The `:alias` parameter (will be assigned to `type`) seems to be optional if any of `number:` `max:` `min:` is specified?

File: cmd/unalias.rb
Comment (Rylan12) [inline cmd/unalias.rb:11]:
```suggestion
```

No need for this if you add `:alias` to the `named_args` method below

File: cmd/unalias.rb
Comment (Rylan12) [reply cmd/unalias.rb]: Adding a type (`:alias`) allows the usage banner to be generated appropriately and automatically

Comment (WavinFlag) [reply cmd/unalias.rb]: Well I see. It is assigned to `@named_args_type`. My fault. 🤦

**Review by Rylan12 — APPROVED** [review]
LGTM! Thanks, @WavinFlag!

@MikeMcQuaid (or someone else) can you merge this when you're able? I don't have write access here.

**Review by MikeMcQuaid — APPROVED** [review]

**Pull request merged**
